src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -94,9 +98,47 @@ public void deletePerson(Person p) {
 	}
 
 	@Override
-
 	public Person createPerson(Person person) {
+		
+		List<Integer> roleIds = new ArrayList<>();
+		List<PersonRole> roleList = daoRole.findAll();
+		System.out.println(roleList);
+				
+		Iterator<PersonRole> iterator = roleList.iterator();
+		
+		while(iterator.hasNext()) {
+			PersonRole role = iterator.next();
+			roleIds.add(role.getId());
+		}
+		
+		System.out.println(roleIds);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS106)